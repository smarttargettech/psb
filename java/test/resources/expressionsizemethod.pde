import processing.pdf.*;

int getWidth() {
  return 400*2;
}

void setup() {
  size(getWidth(), 400/2);
}

void draw() {
  // Draw something good here
  line(0, 0, width/2, height);

  // Exit the program
  println("Finished.");
  exit();
}
