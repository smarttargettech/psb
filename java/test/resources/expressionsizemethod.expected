import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import processing.pdf.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class expressionsizemethod extends PApplet {

public int getWidth() {
  return 400*2;
}

public void setup() {
  size(getWidth(), 400/2);
}

 public void draw() {
  // Draw something good here
  line(0, 0, width/2, height);

  // Exit the program
  println("Finished.");
  exit();
}

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "expressionsizemethod" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
